zigup.zig

```diff
@@ -241,17 +250,12 @@ pub fn main2() !u8 {
             defer allocator.free(installDir);
             const compilerDir = try std.fs.path.join(allocator, &[_][]const u8{ installDir, versionString });
             defer allocator.free(compilerDir);
-            if (std.mem.eql(u8, versionString, "master")) {
-                @panic("set default to master not implemented");
-            } else {
-                try setDefaultCompiler(allocator, compilerDir);
-            }
+            try setDefaultCompiler(allocator, compilerDir);
```

Comment:
I don't think this will work on windows....because of that, I'm not sure we want to support setting the default to "master", or at least, come up with a solution that will work without requiring symlinks like this.

Your change should still work without this particular change, we can create separate Pull Requests for them.  In general, better to keep each pull request focused on one thing.  For this PR, the only change I expect is implementing the clean command itself.

Reply:
Ok I understand. Thanks